src/app/custom/themes/istex-legacy/index.html

```diff
@@ -16,8 +16,8 @@
         <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,600,700" rel="stylesheet" />
         <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet" />
         <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat&amp;subset=latin,latin-ext" type="text/css" />
-        <link rel="stylesheet" href="<%= theme.base.href %>/themes/istex-legacy/css/style.css" />
-        <link rel="stylesheet" href="<%= theme.base.href %>/themes/istex-legacy/css/istex.css?ver2.0" />
+        <link rel="stylesheet" href="<%= theme.base.href %>/themes/istex-legacy/css/style.min.css" />
+        <link rel="stylesheet" href="<%= theme.base.href %>/themes/istex-legacy/css/istex.min.css" />
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏What happened to `?ver2.0` ?

Reply:
I remove it because I don't know why it's here, and the express static files don't handle it, and we only have one file...